lua/neotest-go/patterns.lua

```diff
@@ -1,7 +1,93 @@
+local termcolors = {
```

Comment:
Should these come from a user's config, but default to these values to start, so they can be easily overridden?

Reply:
Yes. What do you think would be the best behavior here;
1. if a user specifies a custom pattern, replace all patterns with the users' patterns
2. if a user specifies a custom pattern, add on to default patterns, and overwrite the default pattern if they have the same name

Reply:
@Davincible I think in most cases plugin authors use `vim.tbl_deep_extend` i.e. we only ever replace something that has been specifically changed by the user so essentially option 2